### PR TITLE
RK-12818 - Add explanation for uWSGI postfork

### PR DESCRIPTION
**Pull request by nathan454**

### CODE REVIEW

File: docs/python-setup.md [hunk 1]
```diff
@@ -225,3 +225,20 @@ If you are using one of those servers, you can set the `fork` argument in the SD
 
 For uWSGI applications, you must enable threads by adding __--enable-threads__ to the command line or __enable-threads = true__ to the uWSGI ini file.  
 You can read more about it [here](https://uwsgi-docs.readthedocs.io/en/latest/WSGIquickstart.html#a-note-on-python-threads).
+
```
Comment (LiranLast) [inline docs/python-setup.md:228]: Let’s make it a separate section- auto fork doesn’t work for uWSGI :)

Comment (lirook) [reply docs/python-setup.md:228]: @LiranLast this is under "### uWSGI applications", it seems to me like the right place to put this. what did you have in mind?

File: docs/python-setup.md [hunk 2]
```diff
@@ -225,3 +225,20 @@ If you are using one of those servers, you can set the `fork` argument in the SD
...
+In addition, you must start rook at each worker seperatly. See sample snippet.
```
Comment (lirook) [inline docs/python-setup.md:229]:
```suggestion
In addition, you must start Rookout at each worker separately using the `postfork` decorator. See the sample snippet below.
```

**Review by lirook — CHANGES_REQUESTED** [review]

File: docs/python-setup.md (hunk 1 above)
Comment (LiranLast) [reply docs/python-setup.md:228]: True that.

Then let’s update the writing to one cohesive copy :)

**Review by lirook — APPROVED** [review]

**Pull request merged**